containers/dev-requirements.txt

```diff
@@ -0,0 +1,14 @@
+-r requirements.txt
+
+nose-capturestderr
+nose-randomly
+coverage
+polib
+pyinotify
+simplejson
+mock
+Sphinx==1.8.5;python_version<="2.7"
+Sphinx>=1.8.5;python_version>="3.0"
```

Comment:
So it seems. I wonder if it would be worth resurrecting the sphinx docs. On the surface the possibility of them seems useful.
However they seem to be rather broken at the moment with the removal of the gtk bits (as the gtk bits are still referenced in the sphinx docs).

Reply:
I'd rather remove the Sphinx completely. It's reintroduction in the future won't be difficult, the setup can create everything itself. We can use the old config in git history as a reference, but the old generated docs we currently have don't seem to be that useful, making it from a scratch sounds quicker.
We should probably create a card to remove all the Sphinx bits, they have been laying in there untouched for years.

Reply:
I agree that we can completely remove this, because nobody uses subscription-manager as Python package and generating  doc string documentation is not useful. We provide D-Bus API, but only for internal use.